package.json

```diff
@@ -33,26 +33,37 @@
   "contributes": {
     "iconThemes": [
       {
-        "id": "glyph.legacy",
+        "id": "glyph.legacy-original",
```

Comment:
hmmm, idk how vscode is going to handle the updating of this. maybe keep it the same or look into how backwards compat works for existing users